src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,18 @@
 package practice;
 
-public class CandidateValidator {
+import java.util.function.Predicate;
+import model.Candidate;
 
+public class CandidateValidator implements Predicate<Candidate> {
+
+    @Override
+    public boolean test(Candidate candidate) {
+        return candidate.getAge() >= 35
```

Comment:
```suggestion
        return candidate.getAge() >= MAX_AGE
```
Let's make constant instead magic numbers